lib/public/Files/Mount/IShareOwnerlessMount.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCP\Files\Mount;
+
+/**
+ * Denotes that shares created under this mountpoint will be manageable by everyone with share permission.
+ *
+ * @since 31.0.0
+ */
+interface IShareOwnerlessMount {
```

Comment:
```suggestion
interface IShareOpenMount {
// interface IShareCollaborativeMount
```

Might be too far and too large a refactor at this point but just throwing in the idea.

Reply:
Hm `IShareCollaborativeMount` doesn't sound too bad. @ArtificialOwl what do you think?

Reply:
I also already did a rename of everything this morning, so doing it once more is not much work :see_no_evil: